lib/beaker/shared/options_resolver.rb

```diff
@@ -0,0 +1,21 @@
+module Beaker
+  module Shared
+    # Methods for parsing options.
+    # - determine if a mode should be run in parallel
+    module OptionsResolver
+      def run_in_parallel?(opts, options, mode)
```

Comment:
yeah, `opts` is definitely used as our local options. I think a better standard would be to actually call it `local_opt(s/ions)`, however.